docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -81,9 +81,9 @@ Some community members also develop on Windows 10 natively with Git for Windows
 
 | Prerequisite                                                                                  | Version | Notes                                                                                       |
 | --------------------------------------------------------------------------------------------- | ------- | ------------------------------------------------------------------------------------------- |
-| [Node.js](http://nodejs.org)                                                                  | `16.x`  | We use the "Active LTS" version, See [LTS Schedule](https://nodejs.org/en/about/releases/). |
+| [Node.js](http://nodejs.org)                                                                  | `18.x`  | We use the "Active LTS" version, See [LTS Schedule](https://nodejs.org/en/about/releases/). |
 | npm (comes bundled with Node)                                                                 | `8.x`   | We use the version bundled with Node.js Active LTS.                                         |
-| [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `4.2.x` | -                                                                                           |
+| [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `4.2.x` | There is no official installion/support for Ubuntu 22.04, we recommend you install with docker with the version mentioned in [Installing Docker](https://contribute.freecodecamp.org/#/how-to-setup-wsl?id=installing-docker-desktop)                                                                                          |
```

Comment:
Hopefully, this changes soon. Also, the formatting for this suggestion might not be correct.

```suggestion
| [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `4.2.x` | MongoDB has not official support for Ubuntu 22.04, you can follow the [Installing Docker Desktop](https://contribute.freecodecamp.org/#/how-to-setup-wsl?id=installing-docker-desktop) step to run MongoDB Community Server within a container |
```